bench/workgen/runner/example_simple.py

```diff
@@ -41,19 +41,22 @@ def show(tname):
     print('<><><><><><><><><><><><>')
     c.close()
 
-context = Context()
-conn = wiredtiger_open("WT_TEST", "create,cache_size=1G")
-s = conn.open_session()
-tname = 'table:simple'
-s.create(tname, 'key_format=S,value_format=S')
+def example_simple_workload(args):
```

Comment:
Can you please add another `example_` function that uses args functionality or comment also should be fine?

Reply:
f0e020f